docs/source/_toctree.yml

```
@@ -1,8 +1,8 @@
 - sections:
   - local: index
     title: Text Generation Inference
-  - local: docker_launch
-    title: Launching with Docker
+  - local: installation_launching
```

The GH Actions workflow is failing here https://github.com/huggingface/text-generation-inference/actions/runs/5812561399/job/15758226898?pr=799

It should be
```suggestion
  - local: installation_launch
```